stable/spinnaker/values.yaml

```diff
@@ -257,11 +257,14 @@ redis:
 minio:
   enabled: true
   image:
-    tag: RELEASE.2018-08-25T01-56-38Z
-  serviceType: ClusterIP
+    tag: RELEASE.2020-01-03T19-12-21Z
+  service:
+    type: ClusterIP
   accessKey: spinnakeradmin
   secretKey: spinnakeradmin
-  bucket: "spinnaker"
```

Comment:
You still need the bucket, otherwise you'll have to change `templates/configmap/halyard-configyaml` to read from the default bucket name. Also, I think the latest value syntax is `defaultBuckets: "spinnaker"`, not a map. Per https://github.com/bitnami/charts/blob/master/bitnami/minio/values.yaml#L160